lib/dialects/oracledb/transaction.js

```diff
@@ -50,40 +50,28 @@ module.exports = class Oracle_Transaction extends Transaction {
     return this.query(conn, `SAVEPOINT ${this.txid}`);
   }
 
-  acquireConnection(config, cb) {
+  async acquireConnection(config, cb) {
     const configConnection = config && config.connection;
```

Comment:
hm
why here configConnection retrieved, but not used in transaction?

Reply:
This is not introduced in pull request, so shouldn't be fixed right now

Reply:
Oh -- it's actually referenced towards the very bottom.  It helps the `Transaction` decide whether or not it should release the connection.

(But, you're actually correct: even though the code is there, it doesn't actually do anything.  It's either leftover from an older feature, or code that was copied unnecessarily)